jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskAdminListPresenterTest.java

```diff
@@ -52,4 +56,11 @@ public void testResumeActionCondition() {
         testTaskStatusCondition(getPresenter().getResumeActionCondition(),
                                 TASK_STATUS_SUSPENDED);
     }
+    
+    @Test
+    public void testReleaseActionOwnerCondition() {
```

Comment:
This test method name is not very informative. Without knowing what problem this PR fixes I wouldn't be able to tell what this method asserts (or to tell without looking at the source code what got broken). How about renaming the test: `public void adminShouldBeAbleToReleaseTaskOwnedByOthers`